modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro

```diff
@@ -566,7 +587,7 @@ definition {
 	}
 
 	@summary = "Default summary"
```

Comment:
@styller1 for maintenance of our code if we had to change into a macro we should set property the  summary description